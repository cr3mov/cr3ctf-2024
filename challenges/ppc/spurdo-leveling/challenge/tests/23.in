119139628151766931 951957528350054877
1 9647254725
1 4738678121
