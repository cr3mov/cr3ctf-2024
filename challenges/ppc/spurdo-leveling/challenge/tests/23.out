57891130 57891130 57891130
